client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/WebHooksService.java

```diff
@@ -4,8 +4,19 @@
 import com.sinch.sdk.core.exceptions.ApiMappingException;
 import com.sinch.sdk.core.utils.databind.Mapper;
 import com.sinch.sdk.domains.numbers.models.v1.webhooks.NumberEvent;
+import java.util.Map;
 
 public class WebHooksService implements com.sinch.sdk.domains.numbers.api.v1.WebHooksService {
+  private final NumbersWebhooksAuthenticationValidation authenticationChecker;
+
+  public WebHooksService(NumbersWebhooksAuthenticationValidation authenticationChecker) {
+    this.authenticationChecker = authenticationChecker;
+  }
+
+  public boolean validateAuthenticationHeader(
```

Comment:
You can add the `@Override` tag to reference the addition to the com.sinch.sdk.domains.numbers.api.v1.WebHooksService interface on line 39

Reply:
👍 Done

Reply:
I can't see it 🧐

Reply:
I can see it 🤩